nautobot/dcim/models/locations.py

```diff
@@ -125,6 +125,22 @@ def get_for_model(self, model):
         content_type = ContentType.objects.get_for_model(model._meta.concrete_model)
         return self.filter(location_type__content_types=content_type)
 
+    @staticmethod
+    def ancestors_without_cte(of, *, include_self=True):
```

Comment:
- Try to merge this with the normal ancestors method
- Put this into the base `TreeQuerySet`